.github/workflows/on-tag-push-create-release.yml

```
@@ -0,0 +1,269 @@
+        echo "Release assets prepared:"
+        ls -la ${{ env.RELEASE_DIR }}/
+
+    - name: Upload prepared release assets
+      uses: actions/upload-artifact@v4
+      with:
+        name: release-assets-${{ needs.common-vars.outputs.tag-name }}
+        path: ${{ env.RELEASE_DIR }}
+
+  create-release:
+    name: Create GitHub Release
+    needs: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)